### PR TITLE
Gui 2425 add time adjustment features from metrics landing, convert to modal directive

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2425
https://eucalyptus.atlassian.net/browse/GUI-2501

### CODE REVIEW

**Comment by aliceh** [comment]
There is a merge conflict in this branch. The CI sequence will only run when there is no merge conflict.

**Comment by dkavanagh** [comment]
CI run failed on buckets: ERROR: Timed out. Did not find element by link_text = 'tmpQclrMD'

hal, retest this please

**Comment by dkavanagh** [comment]
I ran the bucket tests on this branch locally and it failed the 1st time when a bucket-delete dialog wasn't fully populated. It ran fine the 2nd time.

**Comment by aliceh** [comment]
I saw that the bucket test keep s failing in tests in the same place, must be a timing issue I'll look at it. 
Feel free to merge.

File: eucaconsole/templates/dialogs/cloudwatch_chart_dialog.pt [hunk 1]
```diff
@@ -14,32 +16,71 @@
...
+                    <label class="left stack-label" i18n:translate="">Time range</label>
+                    <input id="metric-time-range" type="radio" name="range" value="relative"
+                           ng-model="chartsCtrl.timeRange" /><label i18n:translate="">Relative</label>
+                    <input id="metric-time-range" type="radio" name="range" value="absolute"
+                           ng-model="chartsCtrl.timeRange" /><label i18n:translate="">Absolute</label>
+                </div>
+                <div ng-if="chartsCtrl.timeRange == 'relative'">
+                    <select name="large_chart_monitoring_duration" id="large-chart-monitoring-duration"
+                            class="duration-input"
+                            ng-model="chartsCtrl.largeChartDuration" ng-change="chartsCtrl.handleDurationChange()">
```
Comment (coyote240) [inline eucaconsole/templates/dialogs/cloudwatch_chart_dialog.pt]: I believe tal:repeat allows for tuple unpacking:
(label, value) duration_choices

**Pull request merged**